deploy/ocp-templates/glusterfs-template.yaml

```diff
@@ -35,6 +35,12 @@ objects:
         - image: gluster/gluster-centos:latest
           imagePullPolicy: IfNotPresent
           name: glusterfs
+          QoS Tier:
+            memory: Guaranteed
```

Comment:
@jeremyeder So specify both a CPU and memory `request`, but only specify a `memory` limit?